static/css/devhub/new-landing/base.less

```diff
@@ -162,21 +162,32 @@ h4 {
     text-indent: -9999px;
     margin-right: 4px;
 
-    &.stars-4 { background-position: -13px 0; }
-    &.stars-3 { background-position: -26px 0; }
-    &.stars-2 { background-position: -39px 0; }
-    &.stars-1 { background-position: -52px 0; }
-    &.stars-0 { background-position: -65px 0; }
+    &.stars-5-0 { background-position: -0px 0; }
+    &.stars-4-5 { background-position: -6.5px 0; }
```

Comment:
This isn't going to work - splitting pixels in half doesn't do anything here, but more importantly [the image](https://addons-dev-cdn.allizom.org/static/img/impala/stars.png) doesn't align like that. The whole sprite needs to be redesigned for this approach to work.

Reply:
Oh okay, I thought this might just work

Reply:
I am not that proficient in CSS sorry, it would be better if someone more qualified than me take up on this then.